elastic/apm-server
# add apm-server url to handlebars tmpl

## Motivation/summary

Display a publicly-reachable url for apm-server when running in cloud or eck

closes https://github.com/elastic/apm-server/issues/4895

## Checklist




## How to test these changes

1. Start apm-server under fleet
2. Go to the apm-server integration in the fleet ui
3. Verify that the url is visible
